### ISSUE TITLE
Hide cue arrows on tangible support

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#362]
see https://github.com/phetsims/ratio-and-proportion/issues/142

**Comment by zepumph** [comment]
Added above. This wasn't tested as well as I would have liked because of https://github.com/project-beholder/beholder-detection/issues/4, but it seemed safe to commit. I will test again once that is solved.

**Comment by terracoda** [comment]
I would agree @zepumph, that keyboard interaction cues are not needed for tangible input. The visual cues and the PDOM help text are strictly keyboard dependent. We already have evidence that stumbling upon keyboard interaction cues with a screen reader on a touch device is not at all helpful. The PDOM help text directing learners to use the W/S keys or Arrow keys must not be exposed for tangible input, and even the int to use Lock Ratio interaction will likely not be necessary. 

I guess this will be part of the Self-Voicing design :-)

**Comment by zepumph** [comment]
Unassigning until we add tangible support back into this sim.
